docs/source/en/_toctree.yml

```diff
@@ -44,7 +44,8 @@
       title: Use tokenizers from 🤗 Tokenizers
     - local: multilingual
       title: Inference for multilingual models
-    - sections:
```

Comment:
It's very weird that this gets updated, I copied from main and only added swin2sr, but this gets deleted.

I guess it happens when running `make fix-copies`

Reply:
You should revert this part.